### PR TITLE
feat(backup): use s3 based backup in staging and local

### PR DESCRIPTION
**Pull request by m90**
Ticket https://phabricator.wikimedia.org/T343112

Requires #1080 https://github.com/wbstack/charts/pull/125

### CODE REVIEW

**Comment by m90** [comment]
I tested using a keypair created like this using my local `mc` installation which seemed to work as intended.

File: k8s/helmfile/helmfile.yaml [hunk 1]
```diff
@@ -208,8 +208,8 @@ releases:
 
   - name: wbaas-backup
     namespace: default
-    chart: wbstack/wbaas-backup
-    version: 0.0.6
+    chart: wbastack/wbaas-backup
```
Comment (deer-wmde) [inline k8s/helmfile/helmfile.yaml:211]:
```suggestion
    chart: wbstack/wbaas-backup
```

File: k8s/helmfile/helmfile.yaml [hunk 2]
```diff
@@ -209,7 +209,7 @@ releases:
   - name: wbaas-backup
     namespace: default
     chart: wbstack/wbaas-backup
-    version: 0.0.6
+    version: {{ ternary "0.1.0" "0.0.6" (ne .Environment "production") }}
```
Comment (deer-wmde) [inline k8s/helmfile/helmfile.yaml:212]: missing `.Name` for `.Environment`
```suggestion
    version: {{ ternary "0.1.0" "0.0.6" (ne .Environment.Name "production") }}
```

**Review by deer-wmde — APPROVED** [review]
still works for me locally, excited for staging!

**Pull request merged**